### ISSUE TITLE
AT:  S-9.3 - As a user, I want to have an accessibility route option.

### ISSUE THREAD

**Issue by kajanthyS** [MichelRahme/ConcordiaNavigation#182]
This acceptance test is in reference to issue #113

**Acceptance Criteria** 

**Given** 

- That the directions panel is open 
- that indoor directions are outputted, 
- and that I’ve selected the wheelchair accessible route from the transportation mode bar

**Then** 

- an accessibility route should be displayed on the indoor map.

**Steps** 

1. Click on the search icon 
2. Select the wheelchair icon on the transportation mode bar
3. Input a indoor starting point and destination 
Note: Only floors H1, H8, H9, and MB1 are supported

**Example**
![](https://i.imgur.com/q4qFnKH.gif)

**Comment by Alex-Hayeur** [comment]
Video is recorded, currently editing into gif.

**Comment by mkshiblu** [comment]
ACCEPTED. Well done!
